contributing_to_docs/term_glossary.adoc

```diff
@@ -285,10 +285,16 @@ allowed for use across all distribution documentation:
 ''''
 === OpenShift CLI
 
-Usage: OpenShift CLI
+Usage: OpenShift CLI (`oc`)
 
 The `oc` tool is the command line interface of OpenShift 3 and 4.
 
+When referencing as a prerequisite for a procedure module, use the following
+construction: Install the OpenShift Command-line Interface (CLI), commonly known
```

Comment:
Do we need to put a prerequisite of having the CLI installed in front of every single procedure that uses a CLI command? We certainly don't right now. We should determine when to or not to (if not always).

And if we determine that we should always, then I still think that this is overkill: "Install the OpenShift Command-line Interface (CLI), commonly known as `oc`", as opposed to just "Install the OpenShift CLI (`oc`)" as the prerequisite. Maybe spelling it out more makes sense in places where we know people are just getting started, but it seems a bit much to use for every procedure.

Reply:
I agree that this seems like overkill if we have to add this into every assembly where the `oc` command is used. To me it's the same as saying that we require a running OpenShift cluster before running commands.

I think on the sections that predominantly focus on the CLI, such as some of the CLI reference bucket, it makes sense. Throughout most of the other guides it feels like overkill, though.

Reply:
+1
Installting the CLI is part of the installation, not a specific prerequisite to the task in a given topic.

Reply:
I agree that installing the CLI is part of installation, but then only the person installing OpenShift has the CLI installed, not the person completing that specific task. I think it's fair to assume that the cluster is up but not fair to assume that each user has the CLI installed and the correct permissions to perform an action.

Reply:
I think that we should pair this prereq with another prereq that states the permission you need to run the commands in the procedure.